Editorial changes to MTF Examples (#41)
mulesoft/docs-mule-sdk: modules/ROOT/pages/mtf-parameterized-tests.adoc

= Parameterized Test
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

== Prerequesites

> * The following section assumes that you are familiarized with the parameterization feature in MUnit. +

Reviewer: In a list of prereqs, state the prereq without the leadin (because "Prerequisites" is the leadin):

```suggestion
* You are familiar with the parameterization feature in MUnit. +
```

Reviewer: But wait...there is just one Prerequisite? Hmm...maybe this should just be a plain paragraph under Parameterized Test:

A parameterized suite enables you to.... You must be familiar with this feature in MUnit to benefit from this documentation

Does that work? Is this just a two-sentence topic?

> A parameterized suite allows you to run the same tests with different inputs. See xref:munit::munit-parameterized-suite.adoc[MUnit Parameterized Test Suite] for more information.

Reviewer: See above.

}
----

> You can test different configs as follows:

Reviewer: configs > configurations

</munit:test>
----

== Input Data Parameterization Example

> If you want to execute the same operation but with different inputs, and each of them expecting different outputs, for example, testing an operation that replaces a string input with another:

Reviewer: ```suggestion
Suppose that you want to execute one operation but with different inputs, each with different expected outputs: for example, testing an operation that replaces one string input with another. Your operation uses the following syntax:
```

}
----

> Your test for several inputs by calling the same operation should look as follows:

Reviewer: ```suggestion
Your test for several different inputs to one operation looks like this:
```
